### ISSUE TITLE
Remote Service APIs 

### ISSUE THREAD

**Issue by DCGenomics** [biocaddie/prototype_issues#207]
https://www.ncbi.nlm.nih.gov/home/develop/api.shtml

http://ga4gh.org/#/

**Comment by DCGenomics** [comment]
Also the TCGA CGC stuff

**Comment by nicholsn** [comment]
@jgrethe I just had this question come up as well, is there a RESTful API or clients (R/python) for consuming datamed/biocaddie information?

**Comment by naturalbeau** [comment]
Yes, RESTful API will be available in our new version, which will be released by the end of July, 2017.
